pod/playlist/static/playlist/js/playlist-modal.js

```diff
@@ -1,3 +1,12 @@
+/**
+ * Esup-Pod Playlist modal functions
+ */
```

Comment:
Manque quelques balises :

Exemple pour le fichier `video_dressing.js`
```js
/**
 * @file Esup-Pod functions for the dressing form.
 * @since 3.5.0
 */
```

Reply:
Je veux bien le remplir si tu me trouves depuis quand ce fichier existe ^^

Reply:
V 3.4.0